extern/boostd-data/yugabyte/cassmigrate/migrate.go

```diff
@@ -0,0 +1,111 @@
+package cassmigrate
+
+import (
+	"context"
+	"fmt"
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/yugabyte/gocql"
+	"path"
+	"runtime"
+	"sort"
+)
+
+var log = logging.Logger("migrations")
+
+type migrationFn func(ctx context.Context, session *gocql.Session) error
+
+type migration struct {
+	name string
+	fn   migrationFn
+}
+
+var migrations []migration
+
+func addMigration(fn migrationFn) {
+	_, filename, _, _ := runtime.Caller(1)
+	migrations = append(migrations, migration{
+		name: path.Base(filename),
+		fn:   fn,
+	})
+}
```

Comment:
I think we can just assume the order in which we add the migrations to a slice to the order they should be applied, and just ignore all the sorting and getting of the filename.